src/pages/RecipientLandingScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import {useTranslation} from "react-i18next";
+import LogoFesbalIcon from "../components/icons/LogoFesbalIcon";
+import AppWrapper from "../components/molecules/AppWrapper";
+import {namespaces} from "../i18n/i18n.constants";
+import usersMock from "../mocks/users.mock";
+
+const RecipientLandingScreen = (): JSX.Element => {
+    const {t} = useTranslation(namespaces.pages.recipientLanding);
```

Comment:
Just to improve the readability, could we declare here a variable called "translate" or similar and use it where it is required? For example:

`const {t: translate} = ...`

Reply:
Thanks, fixed @braisgonzalez !